ModbusCommunicator:
    ip: "192.168.1.125"                            
    port : 502                          
